samples/toolbar/src/main/java/org/mozilla/samples/toolbar/SampleToolbarHelpers.kt

```diff
@@ -27,7 +27,7 @@ enum class ToolbarConfiguration(val label: String) {
     PRIVATE_MODE("Private Mode")
 }
 
-@Suppress("MagicNumber")
+@Suppress("MagicNumber", "MemberVisibilityCanBePrivate")
```

Comment:
I wonder why we suppress `MemberVisibilityCanBePrivate` here and can't make the members it's complaining about private?

Reply:
I think you are right. We can do it for `sample-*` packages.